runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1129,6 +1129,12 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          client->write(response, fe->vTableOrITableIndexFromMemberName(comp, std::get<0>(recv)));
          }
          break;
+      case MessageType::VM_delegatingMethodHandleTarget:
+         {  
+         auto recv = client->getRecvData<TR::KnownObjectTable::Index, TR_OpaqueClassBlock *, TR::KnownObjectTable *>();
+         client->write(response, fe->delegatingMethodHandleTarget(comp, std::get<0>(recv), std::get<1>(recv), std::get<2>(recv)));
```

Comment:
If the client created a new index, it needs to be added to the KOT at the server too. Look how we do it in other cases, for instance:
```
      case MessageType::VM_getKnotIndexOfInvokeCacheArrayAppendixElement:
         {
         auto recv = client->getRecvData<uintptr_t *>();
         TR::KnownObjectTable::Index idx = fe->getKnotIndexOfInvokeCacheArrayAppendixElement(comp, std::get<0>(recv));
         client->write(response, idx, knot->getPointerLocation(idx));
         }
```
We write back not only the value of the index, but also the JNI reference that points to the known object obtained with `knot->getPointerLocation(idx)`.
Then, at the server we do the update:
```
   auto recv = stream->read<TR::KnownObjectTable::Index, uintptr_t *>();
   TR::KnownObjectTable::Index idx = std::get<0>(recv);
   knot->updateKnownObjectTableAtServer(idx, std::get<1>(recv));
```

Reply:
okay, thanks. I'm working on it